### PR TITLE
Add method to get the list of languages using in repository

### PR DESCRIPTION
**Pull request by khoa-nd**
Hi kohsuke
I used your API to call Github API but I found that there isn't any method lists all languages using in repository. Therefore, I ask you to add it into your master code.
Thanks

### CODE REVIEW

File: src/main/java/org/kohsuke/github/GHRepository.java [hunk 1]
```diff
@@ -285,6 +285,18 @@ protected void wrapUp(GHTag[] page) {
...
 
+    /**
+     * List languages for the specified repository.
+     * The value on the right of a language is the number of bytes of code written in that language.
+     * {
+         "C": 78769,
+         "Python": 7769
+       }
+     */
+    public Map<String,Integer> listLanguages() throws IOException {
```
Comment (KostyaSha) [inline src/main/java/org/kohsuke/github/GHRepository.java]: Size in bytes, max Integer will be 2Gb, i think some crazy repo may have such number. Long looks better.

**Comment by khoa-nd** [comment]
Hi KostyaSha , 
Thanks for your feedback. I already changed the code as you mentioned

**Comment by KostyaSha** [comment]
Thanks, looks good for me, let's wait @kohsuke

**Comment by oleg-nenashev** [comment]
Merged, thanks!

**Pull request merged**